test/controllers/WhichSubscriptionControllerSpec.scala

```diff
@@ -64,7 +59,7 @@ class WhichSubscriptionControllerSpec extends SpecBase with MockitoSugar with Sc
 
     "return OK and the correct view for a GET" in {
 
-      when(mockProfessionalBodiesService.professionalBodies()).thenReturn(Future.successful(Seq(ProfessionalBody("subscription", List(""),None))))
+      when(mockProfessionalBodiesService.professionalBodies).thenReturn(List(ProfessionalBody("subscription", List(""),None)))
```

Comment:
Can we change these to be an empty list for the synonyms? Having an empty string could be misleading. If we think this is worth the change, there are a few more knocking about the test files that we should change too

```suggestion
      when(mockProfessionalBodiesService.professionalBodies).thenReturn(List(ProfessionalBody("subscription", Nil, None)))
```